<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parameterTab</class>
 <widget class="QWidget" name="parameterTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="centralwidget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{background-color:rgb(235, 235, 235);
color:black}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_pnu_access_title">
          <property name="styleSheet">
           <string notr="true">QLabel {
    border-bottom: 2px solid grey; /* Dunkelgrauer unterer Rand als Linie */
    padding: 3px; /* Ein wenig Abstand zum Text für Ästhetik */
	color: rgb(72, 72, 72);
}
</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;PNU access&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_pnu_title">
            <property name="text">
             <string>PNU:  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_edit_pnu">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
    border: 2px solid #B0BEC5; /* Hellgrauer Rand */
    border-radius: 8px; /* Abgerundete Ecken */
    padding: 4px; /* Innenabstand für Text */
    background-color: white; /* Helle Hintergrundfarbe */
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QLineEdit:hover {
    border-color: #78909C; /* Dunklerer Rand beim Darüberfahren */
}
</string>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="maxLength">
             <number>32767</number>
            </property>
            <property name="cursorPosition">
             <number>0</number>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_pnu_read">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: rgb(0, 170, 255);
    color: white; /* Weiße Schrift */
    border-style: solid;
    border-width: 2px;
    border-radius: 10px; /* Abgerundete Ecken */
    border-color: #0056b3; /* Dunkleres Blau für den Rand */

    min-width: 80px; /* Minimale Breite des Buttons */
    min-height: 30px; /* Minimale Höhe des Buttons */
}

QPushButton:hover {
    background-color: #0056b3; /* Dunkleres Blau beim Darüberfahren */
}

QPushButton:pressed {
    background-color: #004099; /* Noch dunkleres Blau beim Klicken */
}</string>
            </property>
            <property name="text">
             <string>read</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_pnu_feedback">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_pnu_value_title">
            <property name="text">
             <string>value:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_edit_pnu_value">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
    border: 2px solid #B0BEC5; /* Hellgrauer Rand */
    border-radius: 8px; /* Abgerundete Ecken */
    padding: 4px; /* Innenabstand für Text */
    background-color: white; /* Helle Hintergrundfarbe */
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QLineEdit:hover {
    border-color: #78909C; /* Dunklerer Rand beim Darüberfahren */
}
</string>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_pnu_write">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: rgb(0, 170, 255);
    color: white; /* Weiße Schrift */
    border-style: solid;
    border-width: 2px;
    border-radius: 10px; /* Abgerundete Ecken */
    border-color: #0056b3; /* Dunkleres Blau für den Rand */

    min-width: 80px; /* Minimale Breite des Buttons */
    min-height: 30px; /* Minimale Höhe des Buttons */
}

QPushButton:hover {
    background-color: #0056b3; /* Dunkleres Blau beim Darüberfahren */
}

QPushButton:pressed {
    background-color: #004099; /* Noch dunkleres Blau beim Klicken */
}</string>
            </property>
            <property name="text">
             <string>write</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_pnu_value_feedback">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_pnu_list_title">
          <property name="styleSheet">
           <string notr="true">QLabel {
    border-bottom: 2px solid grey; /* Dunkelgrauer unterer Rand als Linie */
    padding: 3px; /* Ein wenig Abstand zum Text für Ästhetik */
	color: rgb(72, 72, 72);
}
</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;PNU list&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_pnu_list_filter_title">
            <property name="text">
             <string>Filter:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_edit_pnu_list_filter">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
    border: 2px solid #B0BEC5; /* Hellgrauer Rand */
    border-radius: 8px; /* Abgerundete Ecken */
    padding: 4px; /* Innenabstand für Text */
    background-color: white; /* Helle Hintergrundfarbe */
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QLineEdit:hover {
    border-color: #78909C; /* Dunklerer Rand beim Darüberfahren */
}

</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="table_view_pnu_list">
          <property name="styleSheet">
           <string notr="true">QTableView {
    border: 1px solid #B0BEC5; /* Grauer Rand für den gesamten View */
    color: #37474F; /* Dunkelgraue Schriftfarbe */
    gridline-color: #B0BEC5; /* Farbe der Gitterlinien */
    background-color: #ECEFF1; /* Hintergrundfarbe */
}

QHeaderView::section {
    background-color: #CFD8DC; /* Hintergrundfarbe der Header */
    padding: 4px;
    border-top: 1px solid #B0BEC5; /* Top-Border der Header */
    border-bottom: 1px solid #B0BEC5; /* Bottom-Border der Header */
    border-right: 1px solid #B0BEC5; /* Right-Border zwischen Headern */
    color: #37474F; /* Schriftfarbe der Header */
    font-weight: bold; /* Fettschrift für die Header-Titel */
    font-size: 12px; /* Schriftgröße */
}

QHeaderView::section:horizontal {
    border-top-width: 0px;
    border-bottom-width: 2px;
    border-right-width: 1px;
}

QHeaderView::section:vertical {
    border-left-width: 0px;
}

QHeaderView::section:horizontal:first, QHeaderView::section:vertical:first {
    border-top-width: 1px; /* Erster Header hat einen Top-Border */
}

QTableCornerButton::section {
    background: #CFD8DC;
    border: 1px solid #B0BEC5;
}

/* Anpassung der Scrollbars */
QScrollBar:vertical {
    background: #ECEFF1;
    width: 10px;
    margin: 15px 0 15px 0;
    border: 1px solid #B0BEC5;
}

QScrollBar::handle:vertical {
    background: #CFD8DC;
    min-height: 20px;
}

QScrollBar::handle:vertical:hover {
    background: #B0BEC5;
}

QScrollBar::sub-line:vertical {
    margin: 3px 0 3px 0;
    height: 10px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar::add-line:vertical {
    margin: 3px 0 3px 0;
    height: 10px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}
</string>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="button_pnu_list_update">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: rgb(0, 170, 255);
    color: white; /* Weiße Schrift */
    border-style: solid;
    border-width: 2px;
    border-radius: 10px; /* Abgerundete Ecken */
    border-color: #0056b3; /* Dunkleres Blau für den Rand */

    min-width: 80px; /* Minimale Breite des Buttons */
    min-height: 30px; /* Minimale Höhe des Buttons */
}

QPushButton:hover {
    background-color: #0056b3; /* Dunkleres Blau beim Darüberfahren */
}

QPushButton:pressed {
    background-color: #004099; /* Noch dunkleres Blau beim Klicken */
}</string>
            </property>
            <property name="text">
             <string>update</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
